pkg/grpc/api.go

```diff
@@ -1499,7 +1503,11 @@ func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigu
 	return nil
 }
 
-func (a *api) UnsubscribeConfigurationAlpha1(ctx context.Context, request *runtimev1pb.UnsubscribeConfigurationRequest) (*runtimev1pb.UnsubscribeConfigurationResponse, error) {
+func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigurationRequest, configurationServer runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server) error { //nolint:nosnakecase
+	return a.SubscribeConfiguration(request, configurationServer.(runtimev1pb.Dapr_SubscribeConfigurationServer))
```

Comment:
is this a type cast and does it work? @shivamkm07 were you able to test it?

Reply:
`runtimev1pb.Dapr_SubscribeConfigurationServer` has the interface function as `runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server` while the type cast happens in the runtime, so would it be better to add these lines to guarantee this by the compiler. In case there is any breaking change in the future, then the code shouldn't compile.
```
func (a *api) SubscribeConfigurationAlpha1(request *runtimev1pb.SubscribeConfigurationRequest, configurationServer runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server) error { //nolint:nosnakecase
	// Currently, the alpha API has the same interface as the beta API, so we add `var _ ...` to guarantee this by the compiler.
	// Once compile fails, `configurationServer.(runtimev1pb.Dapr_SubscribeConfigurationServer)` should be changed then, and this line can be deleted.
	var _ runtimev1pb.Dapr_SubscribeConfigurationServer = runtimev1pb.Dapr_SubscribeConfigurationAlpha1Server(nil)
	return a.SubscribeConfiguration(request, configurationServer.(runtimev1pb.Dapr_SubscribeConfigurationServer))
}
```

Reply:
Done